main.go

```diff
@@ -187,6 +187,16 @@ func main() {
 		klog.ErrorS(err, "unable to create controller", "controller", "NUMAResourcesOperator")
 		os.Exit(1)
 	}
+	if err = (&controllers.KubeletConfigReconciler{
+		Client:    mgr.GetClient(),
+		Scheme:    mgr.GetScheme(),
+		Recorder:  mgr.GetEventRecorderFor("kubeletconfig-controller"),
+		Namespace: namespace,
+	}).SetupWithManager(mgr); err != nil {
+		klog.ErrorS(err, "unable to create controller", "controller", "KubeletConfig")
+		os.Exit(1)
```

Comment:
maybe we can swap all  error logs + os.Exit(1)  with klog.Exitf  , but that's for a different PR. not urgent.

Reply:
note we are using structured logging here (`ErrorS` vs `Exitf`)